docs/Makefile

```diff
@@ -1,67 +1,76 @@
 # Minimal makefile for Sphinx documentation
 #
 
-# You can set these variables from the command line, and also
-# from the environment for the first two.
+# You can set these variables from the command line or the environment
 SPHINXOPTS    ?=
 SPHINXBUILD   ?= sphinx-build
-SOURCEDIR     = .
-BUILDDIR      = _build
-VENV          = .sphinx/venv/bin/activate
+SOURCEDIR     ?= .
+BUILDDIR      ?= _build
+VENVDIR       ?= .sphinx/venv
+IN_VENV       ?= . $(VENVDIR)/bin/activate
 
 
 # Put it first so that "make" without argument is like "make help".
 help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	@echo "The following make targets are available:"
+	@echo "* install   -- set up a virtual-env to work on documentation"
+	@echo "* run       -- watch, build and serve the documentation"
+	@echo "* html      -- build HTML output under $(BUILDDIR)/html"
+	@echo "* serve     -- serve HTML output from $(BUILDDIR)/html"
+	@echo "* epub      -- build epub output under $(BUILDDIR)/epub"
+	@echo "* pdf       -- build pdf output under $(BUILDDIR)/pdf"
+	@echo "* clean-doc -- clean built doc files"
+	@echo "* clean     -- clean full environment"
+	@echo "* spelling  -- check spelling"
+	@echo "* linkcheck -- check external links"
+	@echo "* woke      -- check inclusive language"
+	@echo "-------------------------------------------"
 
 install:
 	@echo "... setting up virtualenv"
-	python3 -m venv .sphinx/venv
-	. $(VENV); pip install --upgrade -r .sphinx/requirements.txt
-
-	@echo "\n" \
-		"--------------------------------------------------------------- \n" \
-		"* watch, build and serve the documentation: make run \n" \
-                "* only build: make html \n" \
-                "* only epub: make epub \n" \
-                "* only serve: make serve \n" \
-                "* clean built doc files: make clean-doc \n" \
-                "* clean full environment: make clean \n" \
-            		"* check spelling: make spelling \n" \
-                "* check links: make linkcheck \n" \
-                "* check inclusive language: make woke \n" \
-		"--------------------------------------------------------------- \n"
+	python3 -m venv $(VENVDIR)
```

Comment:
thought: Should we use a `$(PYTHON)` variable?

Reply:
Hmm, that's quite a neat idea just in case someone wants to use a non-system Python (given everything then occurs in a venv, that's an entirely valid way of working). I'll add it.

Reply:
@waveform80 I added it here: https://github.com/canonical/ubuntu-packaging-guide/pull/16/commits/8c1d61ff90329101b86d2e8fe68f249b35132515

If this is okay, feel free to merge.